chpl/chpl-api/src/main/java/gov/healthit/chpl/web/controller/StatusController.java

```diff
@@ -42,24 +45,27 @@
      * @return JSON object with status of load and "age"
      */
     @ApiOperation(
-            value = "Check if the basic search cache has completed loading. "
-                    + "{ status: 'OK', age: long } is returned if it's finished and "
-                    + "{ status: 'INITIALIZING' } is returned if not. "
-                    + "Age indicates the number of miliseconds since the cache "
-                    + "was last refreshed.",
+            value = "Check the status of every cache. "
+                    + "{ status: 'OK' } is returned if all caches are loaded and "
+                    + "{ status: 'PENDING' } is returned if not. ",
```

Comment:
Is this enough of a change to the API that we need a major version bump? If someone was using "INITIALIZING" for something they'd have to update, right? Would it be better to not change the word, even though the logic behind it is different?

Reply:
I agree - I'll change back to Initializing. I was just going by the language in the ticket.

Reply:
You know as well as I do the language in the ticket is always...umm...never more than 95% correct 😉